client/src/components/Lists/Admin/OrdersList.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react'
+import { getAll } from '../../../api'
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import ListItemText from '@mui/material/ListItemText';
+import Avatar from '@mui/material/Avatar';
+import IconButton from '@mui/material/IconButton';
+import FolderIcon from '@mui/icons-material/Folder';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+export default function OrdersList() {
+  const [orders, setOrders] = useState(null);
+
+  useEffect(() => {
+    getAll('orders').then((data) => {
+      console.log(data)
+      setOrders(data)
+    })
+  }, [])
+
+  return (
+    <List dense>
+      {orders && orders.map((order, index) => {
```

Comment:
MIght be a good idea to check if the array in populated since empty arrays evaluate to true in JS.